docs/man/cdt-cpp.1.md

```diff
@@ -0,0 +1,443 @@
+    
+**`-I=<dir>`**
+    
+    Add directory to include search path
+    
+**`-L=<dir>`**
+
+    Add directory to library search path    
+    
+**`--MF=<file>`**
+
+    Write depfile output    
+
+**`--MD`**
+
+    Write depfile containing user and system headers    
+    
+**`--MMD`**
+    
+    Like -MD except mention only user header files, not system header files. 
+    
+**`--MT=<target>`**
+
+    Specify name of main file output in depfile    
+    
+**`-O=<optimization_level>`**
+    
+    Optimization level s, 0-3     
+    
+**`-R=<dir>`**
+    
+    Add the directory dir to the list of directories to be searched for resource files.
+    
+**`-S`**
+
+    Only run preprocess and compilation steps    
+    
+**`-W=<warning_name>`**
+    
+    Enable the specified warning.   
+    
+**`--abi-version=<string>`**
+    
+    Which ABI version to generate    
+    
+**`--abigen`**
+    
+    Generate ABI
+    
+**`--abigen_output=<string>`**
+    
+    ABIGEN output
+    
+**`--allow-names`**
+    
+    Allows name section to be created
+    
+**`--allow-sse`**
+    
+    Should not be used, except for build libc
+    
+**`--as-secure-log-file-name=<value>`**
+    
+    As secure log file name (initialized from AS_SECURE_LOG_FILE env variable)
+    
+**`--asm-macro-max-nesting-depth=<uint>`**
+    
+    The maximum nesting depth allowed for assembly macros.
+    
+**`-c`**
+    
+    Only run preprocess, compile, and assemble steps
+    
+**`--color`**
+    
+    Use colors in output (default=autodetect)
+    
+**`--contract=<string>`**
+    
+    Contract name
+    
+**`--dD`**
+    
+    Print macro definitions in -E mode in addition to normal output
+    
+**`--dI`**
+    
+    Print include directives in -E mode in addition to normal output
+    
+**`--dM`**
+    
+    Print macro definitions in -E mode instead to normal output
+    
+**`--disable-symbolication`**
+    
+    Disable symbolizing crash backtraces.
+    
+**`--emit-ast`**
+    
+    Emit Clang AST files for source inputs
+    
+**`--emit-ir`**
+    
+    Emit llvm ir
+    
+**`--emit-llvm`**
+    
+    Use the LLVM representation for assembler and object files
+    
+**`--eosio-pp-dir=<string>`**
+    
+    Set the directory for eosio-pp
+    
+**`--fPIC`**
+    
+    Generate position independent code. This option is used for shared libraries
+    
+**`--faligned-allocation`**
+    
+    Enable C++17 aligned allocation functions
+    
+**`--fasm`**
+    
+    Assemble file for x86-64
+    
+**`--fcolor-diagnostics`**
+    
+    Use colors in diagnostics
+    
+**`--fcoroutine-ts`**
+    
+    Enable support for the C++ Coroutines TS
+    
+**`--finline-functions`**
+    
+    Inline suitable functions
+    
+**`--finline-hint-functions`**
+    
+    Inline functions which are (explicitly or implicitly) marked inline
+    
+**`--fmerge-all-constants`**
+    
+    Allow merging of constants
+    
+**`--fnative`**
+    
+    Compile and link for x86-64
```

Comment:
Is this true on an M1/M2 Mac?

Reply:
> Is this true on an M1/M2 Mac?

I don't know. Read my explanation here: https://github.com/AntelopeIO/cdt/pull/139#discussion_r1165400310